server/service/key/key.go

```diff
@@ -6,7 +6,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-func GetCombinedKey(oraclePrivKey []byte, dealID uint64, dataHash []byte) []byte {
+func GetSecretKey(oraclePrivKey []byte, dealID uint64, dataHash []byte) []byte {
```

Comment:
We're going to use the name SecretKey instead of CombinedKey, so we'll also change the function name.